test/dasd_controller_test.rb

```diff
@@ -59,4 +59,13 @@
       expect(Yast::DASDController.Write).to eq(true)
     end
   end
+
+  describe "#ActivateDiag" do
+    it "deactivates and reactivates dasd" do
+      expect(Yast::DASDController).to receive(:DeactivateDisk).once.ordered
```

Comment:
IIRC  that `.once` is not needed, the `expect` by default expects one call. (In contrast to `allow`.)

Reply:
Yes, seem to be so.